website/client/components/tasks/column.vue

```diff
@@ -58,11 +58,50 @@
           @click="openBuyDialog(reward)",
           :popoverPosition="'left'"
         )
+          template(slot="itemBadge", slot-scope="ctx")
+            span.badge.badge-pill.badge-item.badge-svg(
+              :class="{'item-selected-badge': ctx.item.pinned, 'hide': !ctx.highlightBorder}",
+              @click.prevent.stop="togglePinned(ctx.item)"
+            )
+              span.svg-icon.inline.icon-12.color(v-html="icons.pin")
+
 </template>
 
 <style lang="scss" scoped>
   @import '~client/assets/scss/colors.scss';
 
+  .badge-svg {
```

Comment:
this css isn't already available somewhere else?

Reply:
cc @negue

Reply:
@paglias I tried to import them using "@import '/client/components/shops/market/index/badge-svg.css';", however the class is not recognized.

Reply:
`/client/components/shops/market/index/badge-svg.css';` doesn't exist as a file, if you meant the `.vue` file then the css is only available inside that .vue file. You should move it inside a .scss file in client/assets/scss and then you can use it everywhere